packages/salesforcedx-vscode-core/src/commands/forceRenameLightningComponent.ts

```diff
@@ -154,6 +156,31 @@ async function isDuplicate(componentPath: string, newName: string): Promise<bool
   return (allLwcComponents.includes(newName) || allAuraComponents.includes(newName));
 }
 
+/**
+ * check duplicate name under current component directory and __tests__ directory to avoid file loss
+ */
+async function checkForDuplicateInComponent(componentPath: string, newName: string, items: string[]) {
+  let allFiles = items;
+  if (items.includes(TEST_FOLDER)) {
+    const testFiles = await fs.promises.readdir(path.join(componentPath, TEST_FOLDER));
+    allFiles = items.concat(testFiles);
+  }
+  // get all file names and filter out folder names
+  const allFileNames = allFiles.map(file => {
```

Comment:
Is it possible to refactor this to create a named function so the comment can be removed?

Reply:
@randi274 Sounds great! updated the code.